### ISSUE TITLE
form submits when click on ag grid pagination

### ISSUE THREAD

**Issue by felek000** [ag-grid/ag-grid#2138]
[x] bug report => search github for a similar issue or PR before submitting

**Current behavior**
When ag grid wraps around form tag pagination force submit on form.

**Expected behavior**
When ag grid wraps around form tag pagination is not force submit on form.

**Proposed Solution**
Give pagination type="button" atribute.
Example:
```
<button class="ag-paging-button" ref="btNext">&gt;&gt;</button>
<button type="button" class="ag-paging-button" ref="btNext">&gt;&gt;</button>
```

**Minimal reproduction of the problem with instructions**
https://plnkr.co/edit/XmYQwb8nq4qZfV3JeFgx?p=preview
Also don't know why its not loading any data when wraps in form.
 
**What is the motivation / use case for changing the behavior?**

* **ag-Grid version:** X.X.X
most recent, but i think it's same everywere

* **Browser:**
Internet Explorer 11

* **Language:** [ES5]

**Comment by deathwebo** [comment]
+1

**Comment by jakeboone02** [comment]
This is a duplicate of #463, but I have found that the floating filter menu button also has the same issue.  Same fix:  add `type="button"` to the button element.

Looks like it's in the backlog for the next release (see [AG-1073 in the pipeline](https://www.ag-grid.com/ag-grid-pipeline/)), but it's not clear if they're also fixing the floating filter button.

**Comment by petyosi** [comment]
This is addressed in v17.0.0
